trunk/SUAVE/Methods/Aerodynamics/AVL/create_avl_datastructure.py

```diff
@@ -82,7 +82,7 @@ def translate_avl_geometry(geometry):
                 aircraft.append_wing(w)
                 
         for body in geometry.fuselages:
-                if body.tag == 'fuselage':
+                if body.tag == 'fuselage' or body.tag == 'boom':
```

Comment:
We need to stop using tag based processes.

If a geometry needs to be ignored or plotted this should be done in the analysis setup.

Ask me for clarification if you're confused.